datahub-ranger-plugin/conf/ranger-datahub-security.xml

```diff
@@ -0,0 +1,90 @@
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration xmlns:xi="http://www.w3.org/2001/XInclude">
+    <property>
+        <name>ranger.plugin.datahub.policy.rest.url</name>
+        <value>http://[RANGER_SERVICE_HOST]:[RANGER_SERVICE_PORT]</value>
+        <description>
+            URL to Ranger Admin
+        </description>
+    </property>
+
+    <property>
+        <name>ranger.plugin.datahub.service.name</name>
+        <value>datahub</value>
+        <description>
+            Name of the Ranger service containing policies for this datahub instance
+        </description>
+    </property>
+
+    <property>
+        <name>ranger.plugin.datahub.policy.source.impl</name>
+        <value>org.apache.ranger.admin.client.RangerAdminRESTClient</value>
+        <description>
+            Class to retrieve policies from the source
+        </description>
+    </property>
+
+    <property>
+        <name>ranger.plugin.datahub.policy.rest.ssl.config.file</name>
+        <value>ranger-policymgr-ssl.xml</value>
+        <description>
+            Path to the file containing SSL details to contact Ranger Admin
+        </description>
+    </property>
+
+    <property>
+        <name>ranger.plugin.datahub.policy.pollIntervalMs</name>
+        <value>30000</value>
+        <description>
+            How often to poll for changes in policies?
+        </description>
+    </property>
+
+    <property>
+        <name>ranger.plugin.datahub.policy.cache.dir</name>
+        <value>/tmp</value>
+        <description>
+            Directory where Ranger policies are cached after successful retrieval from the source
+        </description>
+    </property>
+
+    <property>
+        <name>ranger.plugin.datahub.policy.rest.client.connection.timeoutMs</name>
+        <value>120000</value>
+        <description>
+            RangerRestClient Connection Timeout in Milli Seconds
+        </description>
+    </property>
+
+    <property>
+        <name>ranger.plugin.datahub.policy.rest.client.read.timeoutMs</name>
+        <value>30000</value>
+        <description>
+            RangerRestClient read Timeout in Milli Seconds
+        </description>
+    </property>
+    <property>
+        <name>ranger.plugin.datahub.service.name</name>
+        <value>ranger_datahub</value>
+        <description>
+            Name of the Ranger service containing policies for this datahub instance
```

Comment:
DataHub, not datahub. Please fix everywhere after doing a search.

Reply:
It is as per Apache Ranger service name convention. It is like an identifier.